cmd/admin-cluster-iam-import.go

```diff
@@ -95,11 +214,13 @@ func mainClusterIAMImport(ctx *cli.Context) error {
 		return nil
 	}
 
-	e = client.ImportIAM(context.Background(), f)
-	fatalIf(probe.NewError(e).Trace(aliasedURL), "Unable to import IAM info.")
-
-	if !globalJSON {
-		console.Infof("IAM info imported to %s from %s\n", aliasedURL, args.Get(1))
+	iamr, e := client.ImportIAMV2(context.Background(), f)
+	if e != nil {
+		e = client.ImportIAM(context.Background(), f)
+		fatalIf(probe.NewError(e).Trace(aliasedURL), "Unable to import IAM info.")
+	} else {
```

Comment:
please test this with previous minio versions, so that this fallback works properly.